projects/Mallard/README.md

```diff
@@ -2,36 +2,12 @@
 
 ## Installing
 
-rn update: run this to clear your cache
-(https://medium.com/@abhisheknalwaya/how-to-clear-react-native-cache-c435c258834e)
-
-```bash
-watchman watch-del-all &&
-rm -rf $TMPDIR/react-native-packager-cache-* &&
-rm -rf $TMPDIR/metro-bundler-cache-* &&
-rm -rf node_modules/ &&
-yarn cache clean &&
-yarn install &&
-yarn start -- --reset-cache
-```
```

Comment:
should't we keep these lines about clearing the packager cache manually?

Reply:
nevermind

Reply:
Yup, I put it at the bottom of the file.